daft/execution/execution_step.py

```diff
@@ -230,6 +230,27 @@ def run(self, inputs: list[vPartition]) -> list[vPartition]:
     def _project(self, inputs: list[vPartition]) -> list[vPartition]:
         [input] = inputs
         return [input.eval_expression_list(self.projection)]
+    
+
+@dataclass(frozen=True)
+class LocalCount(Instruction):
+    partition_id: int
+    logplan: logical_plan.LocalCount
+
+    def run(self, inputs: list[vPartition]) -> list[vPartition]:
+        return self._count(inputs)
+
+    def _count(self, inputs: list[vPartition]) -> list[vPartition]:
+        [input] = inputs
+
+        partition = LocalLogicalPartitionOpRunner()._handle_local_count(
```

Comment:
Once we clean up code in `_handle_local_count` to use `vPartition.count_rows()`, we can use that `vPartition.count_rows()` API here directly as well, which should clean up this `LocalCount` instruction quite a bit.